### DISCUSSION TITLE
Wyze Cam v3 and Thingino - flashing issues

### DISCUSSION THREAD

**Discussion by wes58** [gtxaspec/wz_mini_hacks#793] [Development]
Did anyone have a problem with flashing Thingino firmware on wyze cam 3?
I have 2 cameras which I bought a couple of years ago and I was using wz_mini_hacks.
But I have decided to try Thingino firmware.
Since I only have a windows pc I used windows usbcloner to flash the firmware.
I have followed instructions how to flash the firmware and after a couple of hours of trying I managed to flash the firmware on 1 camera - the problem was getting usb to work, and I still don't know why it worked.
Then I have decided to flash camera 2. No matter what I tried I couldn't do it. It never connected to the camera.
What I tried:
- different USB cables.
- I made a USB cable and connected directly to the socket on the camera PCB
- I checked that there is connection between this socket and ingenic chip.
Nothing worked.
I have decided to make a bootable Ubuntu on a flash drive to try this on Linux.
Trying on Linux still no luck.
I have tried again camera 1 that I managed to flash before and I could read the flash on Ubuntu. 
So the problem is definitely camera number 2.
As the last resort I have decided to swap flash chips and flash 1st camera using flash chip from 2nd camera.
No, problem. Everything worked fine with 1st camera.
After soldering back the flash chip on camera 2, I have now both cameras working. The only issue is that I have to change MAC address, because they both have the same one.
I tried if I can read the flash from camera 2 but it still doesn't work.

So I still have a question what could be the problem with the 2nd camera? Why can't I flash/read firmware (on Ubuntu or Windows)? 
Any suggestions what else I could check before I put the camera back together.

**Comment by gtxaspec** [comment]
do you have discord or telegram? join in on one of them and we can help you tackle it.

**Reply by wes58** [reply]
Hi, 
Unfortunately I don't have any of them.
